generators/client/templates/react/src/main/webapp/app/modules/account/register/register.tsx.ejs

```diff
@@ -32,115 +32,105 @@ export interface IRegisterProps extends StateProps, DispatchProps  {}
 export type IRegisterProps = DispatchProps;
 <%_ } _%>
 
-export interface IRegisterState {
-  password: string;
-}
-
-export class RegisterPage extends React.Component<IRegisterProps, IRegisterState> {
-  state: IRegisterState = {
-    password: ''
-  };
+export const RegisterPage = (props: IRegisterProps) => {
+  const [password, setPassword] = useState('');
 
-  componentWillUnmount() {
-    this.props.reset();
-  }
+  useEffect(() => {
+    return () => props.reset();
```

Comment:
could you change this to `useEffect(() => () => props.reset(), []);` please

Reply:
@chrisdns ok